### DISCUSSION TITLE
Plans to support mingwX64 native target?

### DISCUSSION THREAD

**Discussion by joelightshoe** [evant/kotlin-inject#365] [Q&A]
Are there any plans to support the mingwX64 native target? I'm using kotlin-inject in a library for use in python, and some members of my team use Windows. Thanks!

**Comment by evant** [comment]
Should be easy enough to add to https://github.com/evant/kotlin-inject/blob/main/build-logic/src/main/kotlin/NativeTargets.kt

**Reply by joelightshoe** [reply]
I'll try that, thank you.
